sparta/test/Scoreboard/Scoreboard_test.cpp

```diff
@@ -152,7 +152,7 @@ class ExeUnit : public sparta::Unit
         my_scoreboard_view_.reset
             (new sparta::ScoreboardView(getContainer()->getName(), // ALU0, ALU1, LSU, FPU, etc
                                         sb_unit_type_,             // integer, fp, vector
-                                        getContainer()));          // Used to find the Scoreboard
+                                        getContainer()->getRoot()));          // Used to find the Scoreboard
```

Comment:
_Technically_ you can call `getRoot` here, but this is a tester, no harm.  Also, did it not work with just `getContainer`?

Reply:
`getContainer` didn't work. I think it was too far down the tree, so the scoreboard was above.